Procfile

```diff
@@ -1 +1 @@
-worker: node src/spawn_tests.js
+worker: npm run compile && node lib/main.js test
```

Comment:
Don't we put it in `install` script, see https://github.com/taskcluster/taskcluster-queue/blob/1237170704627ac683b2d4e49275582dd6b51ff7/package.json#L9